src/db/tables/onerep_scans.ts

```diff
@@ -397,6 +398,42 @@ async function getEmailForProfile(onerepProfileId: number) {
   }
 }
 
+async function getScanResultsWithBrokerUnderMaintenance(
+  onerepProfileId: number | null,
+) {
+  if (onerepProfileId === null) {
+    return null;
+  }
+
+  let scanResults = await knex("onerep_scan_results as sr")
+    .select(
+      "sr.*",
+      "s.*",
+      "sr.status as scan_result_status", // rename to avoid collision
+      "db.status as broker_status", // rename to avoid collision
+    )
+    .innerJoin("onerep_scans as s", "sr.onerep_scan_id", "s.onerep_scan_id")
+    .where("s.onerep_profile_id", onerepProfileId)
+    .andWhere("sr.manually_resolved", "false")
+    .andWhereNot("sr.status", "removed")
+    .join("onerep_data_brokers as db", "sr.data_broker", "db.data_broker")
+    .orderBy("sr.onerep_scan_result_id");
+
+  console.log("\n\n scan results broker maintenance:");
+  console.log(scanResults.length);
+
+  scanResults = scanResults.filter(
+    (result) =>
+      result.broker_status === "removal_under_maintenance" ||
+      new Date().getTime() - new Date(result.updated_at).getTime() >
+        200 * 24 * 60 * 60 * 1000,
```

Comment:
If there are a lot of rows, doing this as a `WHERE` clause would be more efficient (assuming `updated_at` column is indexed), and it'd be less data to transfer from the DB to the app as well.

Reply:
I don't see a lot of rows being returned for this particular query. So I thought it's ok to use js for the comparison

Another reason I did it this way is because I think the logic would be actually pretty hard to write out in SQL:
* We want broker status === removal
* For when broker status !== removal, we want to see if `updated_at` is > 200 days